util/string.go

```diff
@@ -0,0 +1,8 @@
+package util
+
+func GetValidParameter(first, second string) string {
```

Comment:
What does it do ?

Reply:
```
	if len(first) > 0 {
		return first
	}
	return second
```

Reply:
I did remove from Ethereum platform because I need reuse that

https://github.com/trustwallet/blockatlas/pull/557/files#diff-ce80ab28ecba3e974a0b3b36b8c16297L8

Reply:
https://github.com/trustwallet/blockatlas/pull/557/files#diff-f240f9e265c26fdb22451440b38e2f58R14